CBLClient/Apps/CBLTestServer-Java-Desktop/build.gradle

```diff
@@ -37,6 +37,9 @@ repositories {
     maven {
         url "https://oss.sonatype.org/content/repositories/snapshots"
     }
+    maven {
```

Comment:
I would have guessed that only one of these two was necessary.  I'm pretty sure that both of those repositories are run by the same org.
... also, I *REALLY* object to using snapshots in our production code (which is what this is: it is not a product, but it is the production test app).  Get a copy and put it on our internal maven server, or something.  Getting snapshots is just way not good.

Reply:
removed the url below. the nanohttpd somehow pointing to snapshots, which has been used in android, did some search, and I am not able to find official nanohttpd product package.

Reply:
Hey @euniceatcouchbase , This really probably is out of scope for this one commit.  Can you do me a favor, though, and open a high-priority ticket to obtain a copy of nanohttpd and to put it on our internal maven server?

Reply:
assign to you? I wasn't able to find. :) sure, I will create a ticket and send you the ticket number

Reply:
No, no no!!! ;-P  Assign to you!   Create a ticket to put nanohttpd in our local maven repo.